uiowa/uiowa
# Feature: Residence hall content type

Resolves https://github.com/uiowa/uiowa/issues/5227

# How to test

- `ddev blt ds --site=housing.uiowa.edu`
- `ddev drush @housing.local en rabbit_hole`
- `ddev drush @housing.local config:import --source ../config/sites/housing.uiowa.edu --partial`
- `ddev drush @housing.local cim`
- `ddev drush @housing.local cr`
- Review content type according to our standards and tell me what I missed
- Verify that all the fields checked off on https://github.com/uiowa/uiowa/issues/5227 are on the content type
- Mockup for content type is at https://sandbox.prod.drupal.uiowa.edu/catlett-hall which helps explain how these fields **might** be used in the design.
